src/pages/Gift/Purchase/Purchaser/index.tsx

```diff
@@ -8,8 +8,8 @@ import { schema } from "./schema";
 import { FormValues } from "./types";
 
 export default function Purchaser({
-  classes = "",
-  wallet,
+  classes: _c = "",
+  wallet: _w,
   ...state
 }: WithWallet<FormStep> & { classes?: string }) {
```

Comment:
```suggestion
}: FormStep) {
```
nit: we could update the props type and remove the need to even access `classes + wallet`

Reply:
I tried this but would need to update parent of `Purchaser`, gift card is stale feature so just keep it as is for now so not to lose context when need to comeback